src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java

```diff
@@ -1,4 +1,4 @@
-//non-compiled with eclipse: extra semicolumn in imports
+// non-compiled with eclipse: extra semicolumn in imports
```

Comment:
please send new  PR to update all usage of no space in this context to have space
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20%22%2F%2Fnon-compiled%22&type=code
no space is required in some Check instances bash scripts.